### ISSUE TITLE
[Japanese] WCAG 3 Introduction 

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wai-translations#232]
I would like to  translate into [Japanese]  the following resource:
- Page URL: [https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/)

I have read the information on Translating WAI Documents at https://www.w3.org/WAI/about/translating/

I will wait for confirmation that the resource is ready for translation.

**Comment by remibetin** [comment]
HI @MakotoUeki Thanks for volunteering to update this resource into Japanese!

I need to apply some updates to this resource before it can be translated. I will let you know when it is done. Should be by next week.

**Comment by remibetin** [comment]
@MakotoUeki The resource has been updated today and is now ready for translation! You can proceed.

**Comment by remibetin** [comment]
Hi @MakotoUeki,

Do you still intend to translate this resource into Japanese?

Let me know if you have any questions or need any help.

**Comment by MakotoUeki** [comment]
@remibetin 
Sorry for the delay. Yes I do. The new working draft will be published in December. Will this document be updated as well? If not, I'll start working on it.

**Comment by remibetin** [comment]
Thank you @MakotoUeki.

> The new working draft will be published in December. Will this document be updated as well?

I will check and get back to you.

**Comment by shawna-slh** [comment]
Updated. I don't expect any major changes for several months. If there are minor changes, I will put them in PRs so they are easy to track.

@MakotoUeki We also made minor updates to [WCAG 2 Overview](https://github.com/w3c/wai-website/pull/1009/files#diff-75f77c87530d05e2dd2cf7e93f7a71ad58362ffacb91ecbca3504bae9295bc41) in case you want to update the [Japanese WCAG 2 Overview translation](https://www.w3.org/WAI/standards-guidelines/wcag/ja)? :)

**Comment by MakotoUeki** [comment]
@shawna-slh 
Got it. Thanks for sharing the updates with me. Let me work on it ASAP.

**Comment by MakotoUeki** [comment]
@remibetin 
I need your help. I forgot about how to proceed the translation work. I just translated the document, but I think I did it in wrong way.....
https://github.com/w3c/wai-website/commit/15b43ba851776bc3c294cecc4d391d67e9713d74

**Comment by remibetin** [comment]
> I need your help. I forgot about how to proceed the translation work. I just translated the document, but I think I did it in wrong way.....
https://github.com/w3c/wai-website/commit/15b43ba851776bc3c294cecc4d391d67e9713d74

@MakotoUeki Overall, this is a good start.

Here is some guidance for you to refine your translation and submit a pull request:
- [How to Translate a WAI Resource](https://www.w3.org/WAI/about/translating/resources/technical-steps/)
- [How to Use GitHub to Translate WAI Resources](https://www.w3.org/WAI/about/translating/resources/using-github/)

If you need my help on any specific steps, please let me know in this issue or at group-wai-translations@w3.org.

**Comment by MakotoUeki** [comment]
@remibetin 
Could you please check this? I'm not sure if I did it right... Thanks a million in advance.
https://github.com/w3c/wai-website/pull/1023

**Comment by remibetin** [comment]
This translation has been published today 🚀 Thank you @MakotoUeki!

See https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/ja

If you happen to spot any issues, feel free to open a new pull request.

**Comment by MakotoUeki** [comment]
@remibetin Domo arigato!
I shared the URL on SNS and many Japanese advocates have already read the translation.

I just found an editorial error and created pull request. Please check it out.
https://github.com/w3c/wai-website/pull/1029
